src/libs/actions/Report.ts

```diff
@@ -543,6 +555,30 @@ function reportActionsExist(reportID: string): boolean {
     return allReportActions?.[reportID] !== undefined;
 }
 
+function updateGroupChatName(reportID: string, reportName: string) {
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {reportName},
+        },
+    ];
+
+    API.write(WRITE_COMMANDS.UPDATE_GROUP_CHAT_NAME, {reportName, reportID}, {optimisticData});
```

Comment:
There's not a whole lot that can go wrong when updating a Group chat name (under normal circumstances). But an error will get set under `report.errorFields.reportName`. Created an issue to address this [here](https://github.com/Expensify/App/issues/39984).